aii-pxelinux/src/main/perl/pxelinux.pm

```diff
@@ -4,12 +4,13 @@ use Sys::Hostname;
 use CAF::FileWriter;
 use CAF::Object;
 use NCM::Component::ks qw (ksuserhooks);
+use AII::Shellfe qw(:pxe_constants);
```

Comment:
Yes, most of them are in fact defined by `aii-shellfe` as provided in this PR...